lib/cc_plugin_imos/cc_plugin_imos/tests/test_imos.py

```diff
@@ -936,7 +935,147 @@ def test_check_acknowledgement(self):
         # check_mandatory_global_attributes
         pass
 
+################################################################################
+#
+# SRS Base Checker
+#
+################################################################################
+
+class TestSRSIMOSBase(unittest.TestCase):
+    # @see
+    # http://www.saltycrane.com/blog/2012/07/how-prevent-nose-unittest-using-docstring-when-verbosity-2/
+    def shortDescription(self):
+        return None
+
+    # override __str__ and __repr__ behavior to show a copy-pastable nosetest name for ion tests
+    #  ion.module:TestClassName.test_function_name
+    def __repr__(self):
+        name = self.id()
+        name = name.split('.')
+        if name[0] not in ["ion", "pyon"]:
+            return "%s (%s)" % (name[-1], '.'.join(name[:-1]))
+        else:
+            return "%s ( %s )" % (name[-1], '.'.join(name[:-2]) + ":" + '.'.join(name[-2:]))
+    __str__ = __repr__
+
+    def load_dataset(self, nc_dataset):
+        '''
+        Return a loaded NC Dataset for the given path
+        '''
+        if not isinstance(nc_dataset, str):
+            raise ValueError("nc_dataset should be a string")
+
+        nc_dataset = Dataset(nc_dataset, 'r')
+        self.addCleanup(nc_dataset.close)
+        return nc_dataset
+
+    def setUp(self):
+        '''
+        Initialize the dataset
+        '''
+        self.srs          = IMOSSRSBaseCheck()
+        self.good_dataset = self.load_dataset(STATIC_FILES['good_data_srs'])
+        self.bad_dataset  = self.load_dataset(STATIC_FILES['bad_data_srs'])
```

Comment:
Again, it would be a bit more obvious if these were just `self.srs_good_data` and `self.srs_bad_data` 
(TBD for the imos tests too...)